src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Abstractions/Channel/TagGroupsEditor.cs

```diff
@@ -0,0 +1,128 @@
+﻿/*
```

Comment:
This file is a move and edit of `AirshipBindings.NETStandard.Abstractions/Push/TagGroupsEditor.cs`. Not sure why GitHub doesn't recognize it as that.